### PR TITLE
Use our error message when raising from Field.coerce

### PR DESCRIPTION
**Pull request by yohanboniface**
We want "Invalid postcode: xxx" instead of "Unable to coerce value for postcode: xxxx".

### CODE REVIEW

File: ban/tests/test_validators.py [hunk 1]
```diff
@@ -174,21 +174,21 @@ def test_cannot_create_postcode_with_code_shorter_than_5_chars(session):
...
-    assert 'code' in validator.errors
+    assert validator.errors['code'] == 'Invalid postcode: "313100"'
 
 
 def test_cannot_create_postcode_with_code_non_digit(session):
     municipality = MunicipalityFactory(insee='12345')
     validator = models.PostCode.validator(code="2A000", name="Montbrun-Bocage",
                                           municipality=municipality)
-    assert 'code' in validator.errors
+    assert validator.errors['code'] == 'Invalid postcode: "2A000"'
```
Comment (davidbgk) [inline ban/tests/test_validators.py]: No Corsica postcodes allowed?

File: ban/core/exceptions.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
+class ValidationError(ValueError):
+    ...
```
Comment (davidbgk) [inline ban/core/exceptions.py]: Heh, I discovered that too recently, still wondering if that's more explicit than the usual `pass`.

Comment (yohanboniface) [inline ban/core/exceptions.py]: I'm in the process of using this as much as possible to extend it's usage and make it as explicit as `pass` :p
Should I refrain? :)

File: ban/tests/test_validators.py (hunk 1 above)
Comment (davidbgk) [inline ban/tests/test_validators.py]: 💣

File: ban/core/exceptions.py (hunk 2 above)
Comment (davidbgk) [inline ban/core/exceptions.py]: I don't know, good to me :)

**Pull request merged**